src/deriver/non_agent/prompts.py

```
@@ -0,0 +1,78 @@
+
+@cache
+def estimate_minimal_deriver_prompt_tokens() -> int:
+    """Estimate base prompt tokens (cached)."""
+    try:
+        prompt = minimal_deriver_prompt(
+            peer_id="",
+            message_created_at=datetime.datetime.now(datetime.timezone.utc),
+            messages="",
+        )
+        return estimate_tokens(prompt)
+    except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)